src/data/markdown/translated-guides/en/03 Results output/100 End-of-test/150-custom-summary.md

```diff
@@ -41,7 +82,227 @@ The keys determine where k6 displays or saves the content:
+    if (key.startsWith('iteration')) delete data.metrics[key];
+  }
+
+  return {
+    stdout: textSummary(data, { indent: '→', enableColors: true }),
+  };
+}
+```
+
+</CodeGroup>
+
+In the collapsible, you can use the tabs to compare default and modified reports.
+
+
+<Collapsible title="Compare default and modified reports" isOpen="" tag="">
+
+Select `Modified` to see the output of the preceding script.
+For compactness, these outputs were limited with the `summaryTrendStats` option.
+
+<CodeGroup labels={["Default report", "Modified"]} lineNumbers={[]} showCopyButton={[true]}>
+
+
+```
+     data_received..................: 63 kB 42 kB/s
+     data_sent......................: 830 B 557 B/s
+     http_req_blocked...............: med=10.39µs  count=5 p(99)=451.07ms p(99.99)=469.67ms
+     http_req_connecting............: med=0s       count=5 p(99)=223.97ms p(99.99)=233.21ms
+     http_req_duration..............: med=202.26ms count=5 p(99)=225.81ms p(99.99)=226.71ms
+       { expected_response:true }...: med=202.26ms count=5 p(99)=225.81ms p(99.99)=226.71ms
+     http_req_failed................: 0.00% ✓ 0        ✗ 5
+     http_req_receiving.............: med=278.27µs count=5 p(99)=377.64µs p(99.99)=381.29µs
+     http_req_sending...............: med=47.57µs  count=5 p(99)=108.42µs p(99.99)=108.72µs
+     http_req_tls_handshaking.......: med=0s       count=5 p(99)=204.42ms p(99.99)=212.86ms
+     http_req_waiting...............: med=201.77ms count=5 p(99)=225.6ms  p(99.99)=226.5ms
+     http_reqs......................: 5     3.352646/s
+     iteration_duration.............: med=204.41ms count=5 p(99)=654.78ms p(99.99)=672.43ms
+     iterations.....................: 5     3.352646/s
+     vus............................: 1     min=1      max=1
+     vus_max........................: 1     min=1      max=1
+```
+
+```
+→    data_received..............: 63 kB 39 kB/s
+→    data_sent..................: 830 B 507 B/s
+→    http_req_blocked...........: med=10.98µs  count=5 p(99)=485.16ms p(99.99)=505.18ms
+→    http_req_connecting........: med=0s       count=5 p(99)=245.05ms p(99.99)=255.15ms
+→    http_req_duration..........: med=208.68ms count=5 p(99)=302.87ms p(99.99)=306.61ms
+→    http_req_failed............: 0.00% ✓ 0        ✗ 5
+→    http_req_receiving.........: med=206.12µs count=5 p(99)=341.05µs p(99.99)=344.13µs
+→    http_req_sending...........: med=47.8µs   count=5 p(99)=166.94µs p(99.99)=170.92µs
+→    http_req_tls_handshaking...: med=0s       count=5 p(99)=207.2ms  p(99.99)=215.74ms
+→    http_req_waiting...........: med=208.47ms count=5 p(99)=302.49ms p(99.99)=306.23ms
+→    http_reqs..................: 5     3.054928/s
+→    vus........................: 1     min=1      max=1
+→    vus_max....................: 1     min=1      max=1
+```
+
+</CodeGroup>
+
+
+</Collapsible>
+
+
+### Custom reports to a remote server
+
+You can also send the generated reports to a remote server (over any protocol that k6 supports).
```

Comment:
1 - Suggest the same:

- ### Example: custom reports to a remote server

2 - The example says `custom reports to a remote server` but it export to 3 outputs with different formats.

I suggest changing this example to showcase only the output to the remote server. 

If not, the title should be different.